static/redfish/v1/JsonSchemas/OemComputerSystem/OemComputerSystem.json

```diff
@@ -196,6 +196,50 @@
                         "array",
                         "null"
                     ]
+                },
+                "ChapData": {
+                    "$ref": "#/definitions/ChapData",
+                    "description": "ChapData.",
+                    "longDescription": "This property shall describe ChapData."
+                }
+            },
+            "type": "object"
+        },
+
+        "ChapData": {
+            "additionalProperties": false,
+            "description": "ChapData.",
+            "longDescription": "This type shall contain properties that describe ChapData.",
+            "patternProperties": {
+                "^([a-zA-Z_][a-zA-Z0-9_]*)?@(odata|Redfish|Message)\\.[a-zA-Z_][a-zA-Z0-9_]*$": {
+                    "description": "This property shall specify a valid odata or Redfish property.",
+                    "type": [
+                        "array",
+                        "boolean",
+                        "integer",
+                        "number",
+                        "null",
+                        "object",
+                        "string"
+                    ]
+                }
+            },
+            "properties": {
+                "ChapName": {
+                    "description": "ChapName.",
+                    "longDescription": "This property shall contain ChapName.",
```

Comment:
Can you use a bit from https://github.com/ibm-openbmc/phosphor-dbus-interfaces/blob/1060/yaml/com/ibm/PLDM/ChapData.interface.yaml#L11 to expand these

Reply:
Done